### PR TITLE
chore(black): upgrade to black 21.4b2

### PR DESCRIPTION
**Pull request by jd**

### CODE REVIEW

File: tests/tracer/test_settings.py [hunk 1]
```diff
@@ -234,7 +234,7 @@ def test_analytics_enabled_attribute(self):
             self.assertFalse(ic.analytics_enabled)
 
     def test_get_analytics_sample_rate(self):
-        """" Check method for accessing sample rate based on configuration """
+        """ " Check method for accessing sample rate based on configuration"""
```
Comment (Kyle-Verhoog) [inline tests/tracer/test_settings.py:237]:
```suggestion
        """ Check method for accessing sample rate based on configuration"""
```

File: tests/tracer/test_settings.py [hunk 2]
```diff
@@ -218,7 +218,7 @@ def test_environment_analytics_enabled(self):
             self.assertEqual(config.foo.analytics_sample_rate, 0.5)
 
     def test_analytics_enabled_attribute(self):
-        """" Confirm environment variable and kwargs are handled properly """
+        """ " Confirm environment variable and kwargs are handled properly"""
```
Comment (Kyle-Verhoog) [inline tests/tracer/test_settings.py:221]:
```suggestion
        """ Confirm environment variable and kwargs are handled properly"""
```

lol I guess we can get rid of these

File: tests/contrib/molten/test_molten.py [hunk 3]
```diff
@@ -29,7 +29,7 @@ def molten_client(headers=None, params=None):
 
 
 class TestMolten(TracerTestCase):
-    """"Ensures Molten is properly instrumented."""
+    """ "Ensures Molten is properly instrumented."""
```
Comment (Yun-Kim) [inline tests/contrib/molten/test_molten.py:32]:
```suggestion
    """Ensures Molten is properly instrumented."""
```

File: tests/contrib/molten/test_molten_di.py [hunk 4]
```diff
@@ -78,7 +78,7 @@ def resolve(self, db: DB) -> Accounts:
 
 
 class TestMoltenDI(TracerTestCase):
-    """"Ensures Molten dependency injection is properly instrumented."""
+    """ "Ensures Molten dependency injection is properly instrumented."""
```
Comment (Yun-Kim) [inline tests/contrib/molten/test_molten_di.py:81]:
```suggestion
    """Ensures Molten dependency injection is properly instrumented."""
```

**Review by Yun-Kim — COMMENTED** [review]
LGTM besides the couple of unnecessary quotation marks in docstrings 😄

File: tests/tracer/test_settings.py [hunk 5]
```diff
@@ -234,7 +234,7 @@ def test_analytics_enabled_attribute(self):
             self.assertFalse(ic.analytics_enabled)
 
     def test_get_analytics_sample_rate(self):
-        """" Check method for accessing sample rate based on configuration """
+        """ Check method for accessing sample rate based on configuration"""
```
Comment (jd) [inline tests/tracer/test_settings.py:237]:
```suggestion
        """Check method for accessing sample rate based on configuration"""
```

Comment (jd) [inline tests/tracer/test_settings.py:237]:
```suggestion
        """Check method for accessing sample rate based on configuration"""
```

**Comment by P403n1x87** [comment]
Just out of curiosity, are we upgrading because we've hit some issues with the previous version of black?

**Comment by jd** [comment]
> Just out of curiosity, are we upgrading because we've hit some issues with the previous version of black?

I don't think we did (yet).

**Review by Yun-Kim — APPROVED** [review]

**Review by Kyle-Verhoog — APPROVED** [review]

**Pull request merged**